city_scrapers/spiders/chi_ssa_27.py

```diff
@@ -0,0 +1,123 @@
+        for item in response_xpath:
+            if "Annual" not in item:
+                meeting = Meeting(
+                    title=self._parse_title(),
+                    description=self._parse_description(),
+                    classification=self._parse_classification(),
+                    start=self._parse_start(item),
+                    end=self._parse_end(),
+                    all_day=self._parse_all_day(),
+                    time_notes=self._parse_time_notes(),
+                    location=self._parse_location(),
+                    links=self._parse_links(),
+                    source=self._parse_source(response),
+                )
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+        yield meeting
+
+    @staticmethod
+    def _parse_title():
+        """Parse or generate meeting title."""
+        meeting_title = "SSA 27 Commission Meeting"
+        return meeting_title
+
+    @staticmethod
+    def _parse_description():
+        """Parse or generate meeting description."""
+        meeting_desc = "Support a vibrant local economy and high quality of life for the " \
+                       "benefit of Lakeview businesses and residents."
+        return meeting_desc
+
+    @staticmethod
+    def _parse_classification():
+        """Parse or generate classification from allowed options."""
+        return NOT_CLASSIFIED
+
+    def _parse_start(self, item):
+        st_datetime = None
+        mon_to_digit = {
+            "jan": 1,
+            "feb": 2,
+            "mar": 3,
+            "apr": 4,
+            "may": 5,
+            "jun": 6,
+            "jul": 7,
+            "aug": 8,
+            "sep": 9,
+            "oct": 10,
+            "nov": 11,
+            "dec": 12,
+        }
+
+        item_elems = item.split()
+        if not item_elems[0].startswith('Annual', 0, 7):
+            month_str = item_elems[0][0:3].lower().rstrip(',')
+            month_digit = mon_to_digit[month_str]
+            yr = int(item_elems[2].rstrip(','))
+            day = int(item_elems[1].rstrip(','))
+            tm = item_elems[3].rstrip(',')
+            hr = int(tm[0:2].rstrip(':'))
+            minit = int(tm[2:4])
+            st_datetime = datetime(yr, month_digit, day, hr, minit)
+            """
+            Parse start datetime as a naive datetime object.
+            """
+        return st_datetime
+
+    @staticmethod
+    def _parse_end():
+        """Parse end datetime as a naive datetime object. Added by pipeline if None"""
+        return None
+
+    @staticmethod
+    def _parse_time_notes():
+        """Parse any additional notes on the timing of the meeting"""
+        return ""
+
+    @staticmethod
+    def _parse_all_day():
+        """Parse or generate all-day status. Defaults to False."""
+        return False
+
+    @staticmethod
+    def _parse_location():
+        """Parse or generate location."""
+        return {
```

Comment:
For static addresses, it's good to add a check to make sure they haven't changed link in this spider https://github.com/City-Bureau/city-scrapers/blob/a863812f1fa3347e854a694672999229e8dbc05c/city_scrapers/spiders/chi_labor_retirement_fund.py#L27